apps/accounts/forms/update.py

```diff
@@ -146,7 +147,10 @@ def __init__(self, instance, *args, **kwargs):
         """Initialization for TimeZoneForm.
         """
         user_profile = instance
+        timezone = [user_profile.timezone for tz in US_TIMEZONE_CHOICES if user_profile.timezone in tz]
+        user_profile.timezone = timezone[0] if timezone else htk_setting('HTK_DEFAULT_TIMEZONE', 'UTC')
```

Comment:
I'm not sure that these two lines are needed. Could you explain why? Please also see the comment below, and address that first, and see if it changes anything

Reply:
Removed the added lines.